lib/framework.js

```diff
@@ -402,22 +393,36 @@ class Framework {
 		return url;
 	}
 
-	processRequests() {
-		this.isPaused = false;
-		this.queue.forEach(function(next) {
-			next();
-		});
-		this.queue = [];
-	}
-
-	pauseRequests() {
-		return (req, res, next) => {
-			if (this.isPaused) {
-				this.queue.push(next);
-			} else {
-				next();
-			}
-		};
+	rewriteUrlBefore(url) {
```

Comment:
It would be great to have a comment for this and `rewriteUrl` explaining their purpose

Reply:
Done